games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild

```diff
@@ -18,8 +18,8 @@ RDEPEND="media-libs/libsdl[joystick,opengl,sound,video]
 	sys-libs/zlib
 	virtual/opengl
 	dev-qt/qtcore:4
-	dev-qt/qtgui:4
-	dev-qt/qtopengl:4
+	dev-qt/qtgui:4[-egl]
+	dev-qt/qtopengl:4[-egl]
```

Comment:
asked virtualjaguar developer.
<damex|work> is it virtualjaguar trouble with egl?
<shamus397> yeah, there's something that needs to be patched to work with egl in the glwidget.h/cpp